docs/api/regressor/peplot.rst

```diff
@@ -5,6 +5,13 @@ Prediction Error Plot
 
 A prediction error plot shows the actual targets from the dataset against the predicted values generated by our model. This allows us to see how much variance is in the model. Data scientists can diagnose regression models using this plot by comparing against the 45 degree line, where the prediction exactly matches the model.
 
+=================   ==============================
+ Visualizer           :class:`~yellowbrick.regressor.residuals.PredictionError`
+ Quick Method         :func:`~yellowbrick.regressor.residuals.prediction_error`
```

Comment:
Did the docs get sorted or is this still not building on yours? I'm happy to pull your branch locally and see what happens when I build it if so.

Reply:
Oh, I just figured out the problem. The issue is that we have the `yellowbrick.regressor.residuals` automodule in both `peplot.rst` and in `residuals.rst` so the documentation doesn't know which one to link to. This doesn't affect your PR, but we should create an issue around it. The solutions that I can see so far are:

1. Move `PredictionError` to `yellowbrick.regressor.peplot` (and associated tests)
2. Combine the residuals documentation into one page (probably not the best)
3. Research a way to get Sphinx to recognize two separate automodule directives (no idea if this is possible)

Reply:
Created #1031 in response.